WorkloadManagementSystem/Agent/StatesMonitoringAgent.py

```diff
@@ -110,13 +99,9 @@ def execute( self ):
         for iP in range( len( self.__summaryValueFieldsMapping ) ):
           rD[ self.__summaryValueFieldsMapping[iP] ] = int( record[iP] )
         rD['time'] = int( Time.toEpoch( now ) )       
-        documents += [rD]
-      res = self.sendRecords( documents, 'WMSHistory' )
-      if res['OK']:
-        gLogger.info( "The records are successfully inserted to MonitoringDB!" )
-      else:
-        #we must use some failover
-        gLogger.error( 'Faild to insert the records: %s', res['Message'] )
+        self.monitoringReporter.addRecord( rD )
+      self.monitoringReporter.commit()
+      gLogger.info( "The records are successfully sent to the Store!" )
```

Comment:
In an agent is preferred to use the provided `self.log` instead of `gLogger` directly